src/components/hmi_message_handler/mock_hmi_message_handler.h

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (c) 2016, Ford Motor Company
```

Comment:
@okozlovlux thanks, I'll fix that, but I think we should just "control-replace" all such places in the project files in a separate pull request.